src/vs/workbench/contrib/positronPlots/browser/modalDialogs/savePlotModalDialog.tsx

```diff
@@ -197,6 +206,9 @@ const SavePlotModalDialog = (props: SavePlotModalDialogProps) => {
 				.then(async (plotResult) => {
 					props.savePlotCallback({ uri: plotResult.uri, path: filePath });
 				})
+				.catch((error) => {
+					props.logService.error('Error saving plot:', error);
```

Comment:
Should this error be displayed to the user?

Reply:
Yes!